ragoon/executors/cde_embedder.py

```diff
@@ -0,0 +1,68 @@
+import random
+
+from datasets import Dataset
+
+from ragoon.executors.embedder import BaseEmbedder
+from ragoon.models.base import Config
+from ragoon.utils import chromadb_normalize_name
+
+
+class CDEEmbedder(BaseEmbedder):
+    def __init__(self, config: Config):
+        import chromadb
```

Comment:
@gkaretka any particular reason why import it in here?

Reply:
If someone decides to implement `BaseEmbedder` without using `chromadb`.